src/components/ComponentConfiguration.stories.tsx

```diff
@@ -1610,7 +1616,7 @@ export const Select: Story = {
       previewSearchInput.focus();
       await userEvent.keyboard('[ArrowDown]');
       await preview.findByRole('listbox');
-      await expect(await preview.findByRole('option', {name: 'Second option'})).toBeVisible();
+      await expect(preview.findByRole('option', {name: 'Second option'})).resolves.toBeVisible();
```

Comment:
that doesn't seem right, but I suppose you're having flakiness here? Can't we instead set a longer timeout for the `findByRole` or wrap it in an `await waitFor(...)`?